### PR TITLE
Anvil tracking update

### PR DESCRIPTION
**Pull request by hanars**
Updates the anvil loading tracker when data is added to a project and when a project is deleted

### CODE REVIEW

**Review by jklugherz — APPROVED** [review]
looks good to me. is it possible to test this manually in dev?

**Comment by hanars** [comment]
> looks good to me. is it possible to test this manually in dev?

yeah it would be!

File: seqr/views/utils/airtable_utils.py [hunk 1]
```diff
@@ -44,15 +47,47 @@ def safe_create_record(self, record_type, record):
...
+            record_type, fields=record_or_filters.keys(), or_filters=record_or_filters, and_filters=record_and_filters,
+            page_size=max_records+1,
+        )
+        if not records or len(records) > max_records:
+            raise ValueError('Unable to identify record to update')
+
+        self._session.params = {}
+        errors = []
+        for record_id in records.keys():
+            response = self._session.patch(f'{self._url}/{record_type}/{record_id}', json={'fields': update})
```
Comment (bpblanken) [inline seqr/views/utils/airtable_utils.py:69]: Definitely a nit, but is there a reason not to put the `patch()` call itself inside the `try/except`?  I think the `patch()` on its own can throw connection errors/timeouts etc. we'd want to continue the loop right?

**Review by bpblanken — APPROVED** [review]
small question but don't want to hold this up!

**Review by bpblanken — APPROVED** [review]

**Pull request merged**